### DISCUSSION TITLE
无法安装，环境如下

### DISCUSSION THREAD

**Discussion by xungegeblue** [w7corp/easywechat#2581] [Q&A]
mac系统、PHP 7.4.23、composer 2.4.1
进入一个空文件夹使用命令composer require overtrue/wechat:~5.0 -vvv ，安装版本v5
无法正常下载安装文件

<img width="668" alt="1" src="https://user-images.githubusercontent.com/14339924/186636142-8fb36d0b-b1c6-4bcf-a82e-880301093844.png">

**Comment by overtrue** [comment]
你从哪里证明无法正常下载安装的？
